schema.sql

```diff
@@ -2,4 +2,3 @@
 
 CREATE TABLE animals (id INT PRIMARY KEY NOT NULL,name TEXT NOT NULL, date_of_birth DATE NOT NULL, escapet_attempts INT NOT NULL, neutured BOOLEAN NOT NULL,weight_kg DECIMAL NOT NULL);
 ALTER TABLE animals ADD COLUMN species TEXT;
```

Comment:
- [OPTIONAL] Consider changing `Text` to `varchar` because, in most circumstances, `varchar` provides better performance, it's more flexible, and can be fully indexed